Kathy.lin/provisioningperformance (#34362)
DataDog/documentation: content/en/gpu_monitoring/_index.md

{{< /callout >}}

## Overview

> Datadog's [GPU Monitoring][1] provides a centralized view into your GPU fleet's health, cost, and performance, regardless of if it's deployed across one of the major cloud providers (AWS, GCP, Azure, Oracle Cloud), hosted on-premises, or provisioned through GPU-as-a-Service platforms like Coreweave and Lambda Labs. 

Reviewer: ```suggestion
Datadog's [GPU Monitoring][1] provides a centralized view into your GPU fleet's health, cost, and performance, whether it's deployed across all of the major cloud providers (AWS, GCP, Azure, Oracle Cloud), hosted on-premises, or provisioned through GPU-as-a-Service platforms like Coreweave and Lambda Labs. 
```
